### PR TITLE
Replace travis with GitHub actions

### PR DESCRIPTION
**Pull request by alex-page**
Replace travis with GitHub actions

### CODE REVIEW

File: .github/workflows/backlog-automation.yml [hunk 1]
```diff
@@ -1,23 +1,25 @@
-name: Automate project columns
+name: Continuous integration
```
Comment (BPScott) [inline .github/workflows/backlog-automation.yml:1]: This gets put into the checks section of the pull request, perhaps we could make that a bit terser

```suggestion
name: CI
```

We should probably rename the file to `ci.yml` too

File: README.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 <div align="center" markdown="1">
 
-[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Travis build status](https://travis-ci.com/Shopify/polaris-tokens.svg?branch=master)](https://travis-ci.com/Shopify/polaris-react)
+[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Continuos integration](https://github.com/shopify/polaris-tokens/workflows/Continuous%20integration/badge.svg)](https://github.com/shopify/polaris-tokens/actions)
```
Comment (alex-page) [inline README.md:11]:
```suggestion
[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Continuous integration](https://github.com/shopify/polaris-tokens/workflows/CI/badge.svg)](https://github.com/shopify/polaris-tokens/actions)
```

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 <div align="center" markdown="1">
 
-[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Travis build status](https://travis-ci.com/Shopify/polaris-tokens.svg?branch=master)](https://travis-ci.com/Shopify/polaris-react)
+[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Continuous integration](https://github.com/shopify/polaris-tokens/workflows/CI/badge.svg)](https://github.com/shopify/polaris-tokens/actions)
```
Comment (alex-page) [inline README.md:11]:
```suggestion
[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![CI](https://github.com/shopify/polaris-tokens/workflows/CI/badge.svg)](https://github.com/shopify/polaris-tokens/actions)
```

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 <div align="center" markdown="1">
 
-[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![Travis build status](https://travis-ci.com/Shopify/polaris-tokens.svg?branch=master)](https://travis-ci.com/Shopify/polaris-react)
+[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![CI](https://github.com/shopify/polaris-tokens/workflows/CI/badge.svg)](https://github.com/shopify/polaris-tokens/actions)
```
Comment (alex-page) [inline README.md:11]:
```suggestion
[![npm version](https://img.shields.io/npm/v/@shopify/polaris-tokens.svg)](https://www.npmjs.com/package/@shopify/polaris-tokens) [![CI](https://github.com/shopify/polaris-tokens/workflows/CI/badge.svg)](https://github.com/shopify/polaris-tokens/actions?query=workflow%3ACI)
```

**Review by BPScott — APPROVED** [review]

**Pull request merged**